packages/lib/shared/services/batch-relayer/batch-relayer.service.ts

```diff
@@ -3,18 +3,37 @@ import {
   EncodeGaugeDepositInput,
   EncodeGaugeMintInput,
   EncodeGaugeWithdrawInput,
+  EncodeJoinPoolInput,
+  EncodeReliquaryCreateRelicAndDepositInput,
+  EncodeReliquaryDepositInput,
 } from './relayer-types'
-import { GaugeActionsService } from './extensions/gauge-actions.service'
+import { gaugeActionsService, GaugeActionsService } from './extensions/gauge-actions.service'
 import { balancerV2BatchRelayerLibraryAbi } from '@repo/lib/modules/web3/contracts/abi/generated'
 import { encodeFunctionData, Hex } from 'viem'
+import {
+  reliquaryActionsService,
+  ReliquaryActionsService,
+} from './extensions/reliquary-actions.service'
+import { vaultActionsService, VaultActionsService } from './extensions/vault-actions.service'
 
 export class BatchRelayerService {
   constructor(
     public readonly batchRelayerAddress: string,
-    private readonly gaugeActionsService: GaugeActionsService
+    private readonly gaugeActionsService: GaugeActionsService,
+    private readonly vaultActionsService: VaultActionsService,
+    private readonly reliquaryActionsService?: ReliquaryActionsService
   ) {}
 
-  public encodePeekChainedReferenceValue(reference: bigint): string {
+  static create(batchRelayerAddress: string, includeReliquary = false): BatchRelayerService {
+    return new BatchRelayerService(
+      batchRelayerAddress,
+      gaugeActionsService,
+      vaultActionsService,
+      includeReliquary ?? false ? reliquaryActionsService : undefined
```

Comment:
```suggestion
      includeReliquary ? reliquaryActionsService : undefined
```
`includeReliquary` is false be default so we don't need `?? false`

Reply:
fixed!